prime-router/docs/design/design/RS-Extention-Registry.yml

```diff
@@ -0,0 +1,3773 @@
+              valueType: String
+              hl7Use: EI.4
+    EI:
+      Example:
+        "identifier": [
+          {
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
+                "valueString": "EI.2"
+              }
+            ],
+            "value": "NAME"
+          },
+          {
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
+                "valueString": "EI.3,EI.4"
+              }
+            ],
+            "type": {
+              "coding": [
+                {
+                  "extension": [
+                    {
+                      "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/codeable-concept-id",
+                      "valueBoolean": true
+                    }
+                  ],
+                  "code": "ISO"
+                }
+              ]
+            },
+            "value": "UNI"
+          },
+          {
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field",
+                "valueString": "ORC.2"
+              },
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/assigning-authority",
+                "extension": [
+                  {
+                    "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/namespace-id",
+                    "valueString": "Placer Identifier Namespace"
+                  },
+                  {
+                    "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id",
+                    "valueString": "Placer Universal ID"
+                  },
+                  {
+                    "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/universal-id-type",
+                    "valueCode": "ISO"
+                  }
+                ]
+              },
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/ei-assigner-organization-type",
+                "valueString": "default"
+              }
+            ],
+            "value": "Entity ID",
+            "assigner": {
+              "reference": "Organization/1707847155713462000.da8053be-d79a-4c15-9e50-ec19218d7287"
+            }
+          },
+          {
+            "extension": [
+              {
+                "url": "https://reportstream.cdc.gov/fhir/StructureDefinition/ei-assigner-organization-type",
+                "valueString": "organization"
+              }
+            ],
+            "value": "Entity ID",
+            "assigner": {
+              "reference": "Organization/1707769195125082000.40e4d3f4-bbd6-43af-aefe-5894b31e9806"
+            }
+          }
+        ]
+      Extension:
+        - id: ei-assigner-organization-type
```

Comment:
Not sure I have a great solution, but this was a tad confusing.  This extension is one where the value is either "default" or "organization"?  Maybe they should be collapsed and it should be `constants`?